### DISCUSSION TITLE
Why is geekbench slower for me?

### DISCUSSION THREAD

**Discussion by hello-smile6** [copy/v86#698] [General]
I ran geekbench and got these results:
https://browser.geekbench.com/v3/cpu/9003804

However, https://github.com/copy/v86/discussions/542#discussion-3647837 shows far better results. Did something change recently, or is my computer just slower?

**Comment by copy** [comment]
> or is my computer just slower

Probably slower or throttling. You may also want to try a different browser.

**Comment by spetterman66** [comment]
my friend has done a Geekbench 4 in arch32 emulated inside v86, here's the result: https://browser.geekbench.com/v4/cpu/16858165

**Comment by ProgrammerIn-wonderland** [comment]
Your CPU single core performance might just be lower. v86 is absolutely abysmal on my school laptop with a Pentium gold but great on my i7-8700 for example
